presto-docs/src/main/sphinx/release/release-0.227.rst

```diff
@@ -0,0 +1,39 @@
+=============
+Release 0.227
+=============
+
+General Changes
+_______________
+* Enable spatial joins (broadcast and distributed) for ST_Equals, ST_Overlaps, ST_Crosses, and ST_Touches.
+* Fix a bug where index joins would fail with the error "driver should never block".  Now queries will wait until a configured timeout for the indexes to load.  The timeout can be configured with the configuration property ``index-loader-timeout`` and the session property ``index_loader_timeout``.
+* Move ``AggregationNode`` to SPI. Connectors can now push down aggregation to table scan.
+* Move ``ProjectNode`` to SPI. Connectors can now push down project to table scan.
+* Add ``task_partitioned_writer_count`` session property to allow to set number of concurrent writers for partitioned (bucketed) writes independently.
+* Removing syntax support for ``SET PATH``, ``CURRENT_PATH`` (Presto server do not use this information)
```

Comment:
```
Removing syntax support for ``SET PATH`` and ``CURRENT_PATH``. Those information was never used by Presto.
```

Reply:
@caithagoras Thanks for the improvement! I additionally modified it to
```
Remove syntax support for ``SET PATH`` and ``CURRENT_PATH``. This information was never used by Presto.
```
Removing -> Remove
Those information -> This information

Reply:
What about `The path information`

Reply:
> What about `The path information`
@caithagoras 
It sounds like all path information was not used which I'm not sure. 

We have a few choices here, which one do you prefer?
```
Remove syntax support for ``SET PATH`` and ``CURRENT_PATH``, which was
  never used by Presto.
```
or
```
Remove syntax support for ``SET PATH`` and ``CURRENT_PATH``. Such information was
  never used by Presto.
```
or
```
Remove syntax support for ``SET PATH`` and ``CURRENT_PATH``. This information was
  never used by Presto.
```

Reply:
@caithagoras can you look at this one?